package.json

```diff
@@ -162,7 +162,7 @@
     "react-native-launch-arguments": "^4.0.2",
     "react-native-localize": "^2.2.6",
     "react-native-modal": "^13.0.0",
-    "react-native-onyx": "2.0.86",
+    "react-native-onyx": "2.0.87",
```

Comment:
It looks like we bump the onyx version, is there any breaking changes that I need to verify